polls/templates/polls/htmlform.html

```diff
@@ -9,24 +9,38 @@ <h3>Task</h3>
   <input type="hidden" value="{{ task_list }}" name="tasklist">
   <input type="hidden" value="{{ task_presentation }}" name="taskPresentation">
   <h4>ID: {{task.pk}}</h4>
-  <audio controls src="{{ reference_url }}" class="mt-3">
-    Your browser does not support the
-    <code>audio</code> element.
-  </audio>
-  <audio controls src="{{ transform_url }}" class="mt-3">
-    Your browser does not support the
-    <code>audio</code> element.
-  </audio>
-
-  <div class="mt-2">
-    <div class="form-check form-check-inline">
-      <input class="form-check-input" type="radio" id="type1" name="annotationOption" value="XXY" checked>
-      <label class="form-check-label" for="type1">First and Second is same</label>
+  <div>
+    {% for audio in audios %}
+
+    {% if forloop.counter == 1 %}
+    <div>
+
+      <audio controls src="{{ audio }}" class="mt-3">
+        Your browser does not support the
+        <code>audio</code> element.
+      </audio>
     </div>
-    <div class="form-check form-check-inline">
-      <input class="form-check-input" type="radio" id="type2" name="annotationOption" value="XYX">
-      <label class="form-check-label" for="type2">First and Third is same</label>
+
+    {% else %}
+    <audio controls src="{{ audio }}" class="mt-3">
+      Your browser does not support the
+      <code>audio</code> element.
+    </audio>
+
+    {% endif %}
+
+
+    {% endfor %}
+  </div>
+
+  <div class="mt-2 d-flex justify-content-around">
+    {% for annotation in task_annotations %}
+    <div class="form-check form-check-inline mx-10">
+      <input class="form-check-input" type="radio" id="type1" name="annotationOption"
+        value="{{ annotation.annotation }}">
     </div>
+    {% endfor %}
+
   </div>
 
```

Comment:
We can as well have hx-target="body" in the submit button tag, so the new page sent by the django views would be rendered on the body tag in base.

Reply:
I think I may have a better approach for this. How about we just create a parent div and include the html returned by view in it. I can that will make the code more clean and lean